cardinal/system/system_tokeniser.go

```diff
@@ -118,46 +118,45 @@ func (ts *TokeniserSystem) setupGrammar() {
 // FishTokens processes the tokenized command and returns VerbData
 func (ts *TokeniserSystem) FishTokens(tokens []string) component.VerbData {
 	var data component.VerbData // Initialize VerbData to store the result
-	var err uint8 = 0           // Error code variable
 	lenTokens := len(tokens) - 1
 
 	// Look up the verb, object, and directional object from the tokens
-	var vrb enums.ActionType = ts.cmdLookup[(tokens[0])]
-	var obj enums.ObjectType = ts.objLookup[(tokens[lenTokens])]
-	var dobj enums.DirObjectType = ts.dirObjLookup[(tokens[lenTokens])]
+	var VRB enums.ActionType = ts.cmdLookup[(tokens[0])]
```

Comment:
I think idiomaticaly it should be Vrb as the ones below

Reply:
done